refactor(admin): 2FA chapter clean-up (#12446)
nextcloud/documentation: admin_manual/configuration_user/two_factor-auth.rst

=========================

Two-factor authentication adds an additional layer of security to user accounts. In order to log

> in on an account when two-factor authentication (2FA) enabled, it is necessary to provide both the

Reviewer: it is necessary to provide -> you must provide
(direct language is often easier to understand)

Reviewer 2: ```suggestion
in on an account when two-factor authentication (2FA) enabled, you must provide both the
```

login password and another factor. 

To use 2FA two things must happen:

- At least one 2FA provider must be enabled by the administrator.
- A user must activate 2FA on their account (or) the administrator must enforce the use of 2FA.

Both steps are described below.

Enabling two-factor authentication
----------------------------------

2FA in Nextcloud is pluggable, meaning that various 2FA providers can be used to support different 

> types of factors. Three providers are automatically installed (but may need to be enabled):

Reviewer 2: Its not preinstalled so covered by "others can be found in the app store"

 sudo -u www-data php occ twofactorauth:cleanup <provider_id>

> .. warning:: This operation is irreversible. Only run it for providers you do not intend to enable again.

Reviewer: Sound a bit like you can never use it again. It's just the current settings are all lost, and if you enable it you have to set up everything again (might be interesting for people who have messed up something with their setup)?

> Disabling two-factor authentication

Reviewer: For me the use case is, that a user has forgotten/lost their second factor. If there are different factors possible, I'd add a line to show 2FA status of the user first:
`sudo -u www-data php occ twofactorauth:state <uid>`
